packages/migrations/src/addDappnodePeerToIpfsNodes.ts

```diff
@@ -0,0 +1,26 @@
+import { logs } from "@dappnode/logger";
+
+//Adds Dappnode IPFS peers to the local IPFS node for more stability when fetching IPFS content
+export async function addDappnodePeerToIpfsNodes() {
+  const nodesIps = [
+    "167.86.114.131", //local node
+    "65.109.51.31", //production node
+  ];
+
+  const ipfsAlias = "ipfs.dappnode";
+  const dappnodeIpfsPeer = (ipfsNodeIp: string) =>
+    `/ip4/${ipfsNodeIp}/tcp/4001/p2p/QmfB6dT5zxUq1BXiXisgcZKYkvjywdDYBK5keRaqDKH633`;
```

Comment:
Are you sure it works like this? each IPFS node has a different ID, so production and dev nodes have a different node ID 

Checkout https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-swarm-peering-add